### DISCUSSION TITLE
io.minio.errors.ErrorResponseException: Access denied，only minioClient.bucketExists API

### DISCUSSION THREAD

**Discussion by Garc-du** [minio/minio#17507] [Q&A] [answered]
When I use nginx proxy minio, whether or not I use a domain name or IP direct proxy access, this "io.minio.errors.ErrorResponseException: Access denied" problem appears, because it is on the same server, so there is no time or network policy. I use the same nginx configuration on my local computer, which can be accessed and passed normally, but nginx on the server still reports this error. The nginx version is the same.
`  
	upstream minio {
		server 127.0.0.1:9000;
		server 127.0.0.1:9000;
	}
server {
        listen 80;
        server_name www.yini.com;

	location / {
			proxy_set_header Host $host;
			proxy_pass http://minio;
			proxy_set_header  X-Real-IP    $remote_addr;
                        proxy_set_header  X-Forwarded-For $proxy_add_x_forwarded_for;
		}
	}
`
I have tried both http and https

`file. minio.endpoint: 'http://www.yini.com' `

The weirder problem is that I use the "MinioClient.builder() .endpoint(endpoint,80,false).credentials(accessKey, secretKey) .build()" on the API。I can access it successfully the first time and prompt this error the second time.
However, if you remove the two parameters of port number and secure, it will not be accessible at all and will report an error

`
JKD17
minio version: 8.5.3
docker
nginx 1.22.1
`

**Comment by Garc-du** [comment, marked as answer]
I found the reason, but this reason puzzled me. I only have problems when using the minioClient.bucketExists API, and all other APIs can be read and written normally

**Reply by missuo** [reply]
same issue
